test/scripts/integration_tests.py

```diff
@@ -82,11 +117,20 @@ def setUp(self):
         Ensure the custom resource definition exists, and set up the status and metrics url.
         """
         self.__kubectl = Kubectl()
-
+        # Set up the tiller and expose it via a nodeport service
+        self.__helm = Helm()
+        self.__helm.init()
+        self.__kubectl.apply(_TEST_DATA_DIRECTORY + "/tiller_nodeport_service.yml")
+        # Ensure the CRD is there and there are no characters, for a clean starting point
+        self.__kubectl.apply(_CUSTOM_RESOURCE_DEFINION_FILE)
```

Comment:
I believe this will break the testing in cases where it is expecting a certain number of events. This was worked around by using a make target to run the tests. Potentially that same make target can be used here, or we can figure out a better way to do the testing.

Reply:
I dont think this will break because this happens before the lostromos process is started, and lostromos doesn't watch the CRD resource directly, only instances of it.  Right now without this it breaks running integration tests locally because the CRD set up is only in the travis config.